server/services/licenceService.ts

```diff
@@ -639,8 +639,8 @@ export default class LicenceService {
   private async getCroNumberFromNomis(prisonerNumber: string, user: User): Promise<string> {
     const prisoners = await this.prisonerService.searchPrisonersByNomisIds([prisonerNumber], user)
     if (prisoners) {
-      return prisoners[0]?.croNumber
+      if (prisoners[0]?.croNumber !== undefined) return prisoners[0]?.croNumber
```

Comment:
you can make this a bit less verbose using `||`

`return prisoners[0]?.croNumber || ''`